docs/serving/samples/multi-container/README.md

```diff
@@ -0,0 +1,250 @@
+   COPY . ./
+   # Build the binary.
+   # -mod=readonly ensures immutable go.mod and go.sum in container builds.
+   RUN CGO_ENABLED=0 GOOS=linux go build -mod=readonly -v -o servingcontainer
+   # Use the official Alpine image for a lean production container.
+   # https://hub.docker.com/_/alpine
+   # https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds
+   FROM alpine:3
+   RUN apk add --no-cache ca-certificates
+   # Copy the binary to the production image from the builder stage.
+   COPY --from=builder /app/servingcontainer /servingcontainer
+   # Run the web service on container startup.
+   CMD ["/servingcontainer"]
+   ```
+
+### Sidecar Container
+1. Navigate to cloned code directory `cd -`
+1. `cd knative-docs/docs/serving/samples/multi-container/sidecarcontainer`
+1. Update a file named `sidecarcontainer.go` and copy the code block below into it. This
+   code creates a basic web server which listens on port 8882:
+
+   ```go
+   package main
+   import (
+   	"fmt"
+   	"log"
+   	"net/http"
+   )
+   func handler(w http.ResponseWriter, r *http.Request) {
+   	log.Println("sidecar container received a request.")
+   	fmt.Fprintln(w, "Yay!! multi-container works")
+   }
+   func main() {
+   	log.Print("sidecar container started...")
+   	http.HandleFunc("/", handler)
+   	log.Fatal(http.ListenAndServe(":8882", nil))
+   }
+   ```
+
+1. In your project directory, update a file named `Dockerfile` and copy the code
+   block below into it. For detailed instructions on dockerizing a Go app, see
+   [Deploying Go servers with Docker](https://blog.golang.org/docker).
+
+   ```docker
+   # Use the official Golang image to create a build artifact.
+   # This is based on Debian and sets the GOPATH to /go.
+   # https://hub.docker.com/_/golang
+   FROM golang:1.13 as builder
+   # Create and change to the app directory.
+   WORKDIR /app
+   # Retrieve application dependencies using go modules.
+   # Allows container builds to reuse downloaded dependencies.
+   COPY go.* ./
+   RUN go mod download
+   # Copy local code to the container image.
+   COPY . ./
+   # Build the binary.
+   # -mod=readonly ensures immutable go.mod and go.sum in container builds.
+   RUN CGO_ENABLED=0 GOOS=linux go build -mod=readonly -v -o sidecarcontainer
+   # Use the official Alpine image for a lean production container.
+   # https://hub.docker.com/_/alpine
+   # https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds
+   FROM alpine:3
+   RUN apk add --no-cache ca-certificates   
+   # Copy the binary to the production image from the builder stage.
+   COPY --from=builder /app/sidecarcontainer /sidecarcontainer   
+   # Run the web service on container startup.
+   CMD ["/sidecarcontainer"]
+   ```
+
+### Writing Knative Service YAML
+
+1. Go to multi-container directory where `service.yaml` already exist
+   1. `cd -`
+   2. `cd knative-docs/docs/serving/samples/multi-container/`
```

Comment:
Something like:

1. After you have cloned the sample repository, navigate to the `multi-container` directory:
   ```
   cd knative-docs/docs/serving/samples/multi-container/
   ```